provider/gke/gke.go

```diff
@@ -260,7 +260,7 @@ func (c *GKE) nodePoolCreated(req *containerpb.CreateNodePoolRequest) (bool, err
 		return false, err
 	}
 	log.Printf("cluster node pool status: `%v`", rep.Status)
```

Comment:
now looking at this again should we make check what is the  rep.Status and compare it to running or created?